util/CosmosHelper.js

```diff
@@ -111,7 +122,8 @@ export async function getTransferInfo(txHash, opt) {
       _amount: amount,
     };
   }
-  const isFailed = (code && code !== '0') || !success;
+  const fullLogs = JSON.parse(rawLog.slice(1, rawLog.length - 1));
```

Comment:
Why use `.slice()`? If `rawLog` is an JSON array, we should parse full string and use `fullLogs[0]` below

Reply:
oh! I misunderstood that JSON.parse can only be used in object.